sign/schnorr/schnorr_test.go

```diff
@@ -65,34 +62,3 @@ func TestEdDSACompatibility(t *testing.T) {
 	}
 
 }
-
-// Simple random stream using the random instance provided by the testing tool
-type quickstream struct {
-	rand *rand.Rand
-}
-
-func (s *quickstream) XORKeyStream(dst, src []byte) {
-	s.rand.Read(dst)
-}
-
-func (s *quickstream) Generate(rand *rand.Rand, size int) reflect.Value {
-	return reflect.ValueOf(&quickstream{rand: rand})
-}
-
-func TestQuickSchnorrSignature(t *testing.T) {
-	f := func(rand *quickstream, msg []byte) bool {
-		suite := edwards25519.NewBlakeSHA256Ed25519WithRand(rand)
-		kp := key.NewKeyPair(suite)
-
-		s, err := Sign(suite, kp.Private, msg)
-		if err != nil {
-			return false
-		}
-
-		return Verify(suite, kp.Public, msg, s) == nil
-	}
-
-	if err := quick.Check(f, nil); err != nil {
-		t.Error(err)
-	}
-}
```

Comment:
i think this is a sign of a bad merge, I remember reviewing this code but here it got removed.

Reply:
I agree. The problem I'm having is that somehow the commits you were building on have two parents, but are not merges. I'll try again a different way and see what I get.